addons/mail/static/src/core/common/thread.xml

```diff
@@ -60,7 +60,14 @@
                         <h4 class="mb-3 fw-bolder">
                             Congratulations, your inbox is empty
                         </h4>
-                        New messages appear here.
+                        <div t-if="!store.user.notificationByInbox" class="align-items-center text-center">
+                            <button class="btn btn-link m-0 px-1 py-0 align-baseline" t-on-click="onClickPreferences">
+                                Change your preferences to receive new notifications in your Inbox.
+                            </button>
+                        </div>
```

Comment:
I know we talked about this, but why not put that in `_t("<button>Change your preferences</button> to receive new notifications in your Inbox.")`, and have template build pre-text/button/post-text from translated string?

I get this is more LOCs and looks pretty unconventional in some other parts in Odoo... Usually it has this nice style in English but other languages are stuck with exact same shape for partial link in sentence. I think it's a poor compromise to make implementation simpler by putting everything in a button link...

So I really prefer partial button link and making some processing in JS/template.